cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -8,59 +8,52 @@
 
    Creates Sidebar Breakout markup when given:
 
-   settings:               An object used to customize the markup.
-
-   settings.heading:       A string containing heading text.
+   sidebar:                An object used to customize the markup.
 
-   settings.image:         An object used to customize the image
-                           markup.
+   sidebar.heading:        A string containing heading text. ()
 
-   settings.intro.body:    A string containing body text.
+   sidebar.image.url:      A string containing the url for the image. (Optional)
 
-   settings.intro.heading: A string containing heading text.
+   sidebar.image.is_round: A Boolean indicating whether the image is round. (Optional)
 
-   settings.related_posts: An object used to customize the related posts
-                           markup.
+   sidebar.intro.heading:  A string containing heading text. (Optional)
 
-   settings.text_intro:    The text-introduction molecule markup that can
-                           be passed in instead of using the Jinja call
-                           method.
+   sidebar.intro.body:     A string containing body text. (Required)
 
-   ========================================================================== #}
+   ========================================================================= #}
```

Comment:
The heading, image, intro heading, intro body and related posts are all optional in the breakout sidebar below. They should also be optional here with condition wrappers in the code in case they aren't used.